Quick_Deploy/HuggingFaceTransformers/README.md

```diff
@@ -26,125 +26,155 @@
+    {
+      "name": "text",
+      "datatype": "BYTES",
+      "shape": [
+        1
+      ],
+      "data": [
+        "I am going to be in the market for a new laptop soon. I"
+      ]
+    }
+  ]
+}
 ```
-*NOTE*: You may need to install packages for the client to run. See [client.py](client.py)
-for details and run the script in an environment manager if you are concerned about 
-version collisions.
 
-In the case of this tutorial, we have included models that are specialized for one of 
-two tasks: text classification or text generation. The ```distilbert-base-uncased-finetuned-sst-2-english```
-model is specialized for text classification and we can query the model in the following way:
+## Step 5: Host Multiple Models in Triton
 
-```bash
-python3 client.py -m distilbert-base-uncased-finetuned-sst-2-english --prompt "I feel great!"
-```
+So far in this tutorial, we have only loaded a single model. However, Triton is capable
+of hosting many models, simultaneously. To accomplish this, first ensure you have
+exited the docker container by invoking `Ctrl+C` and waiting for the container to exit.
 
-Which will return a result similar to:
+Next copy the remaining models provided into the model repository:
+```
+cp -r mistral7b/ model_repository/
+cp -r persimmon8b/ model_repository/
 ```
-| Name        | Shape       | Data              |
-| ----------- | ----------- | ----------------- |
-| prompt      | [1]         | ['I feel great!'] |
-| label       | (1,)        | POSITIVE          |
-| score       | (1,)        | [0.99986434]      |
+*NOTE*: The combined size of these three models is large. If your current hardware cannot
+support hosting all three models simultaneously, consider copying only a single additional
+model.
+
+Again, launch the server by invoking the `docker run` command from above and wait for confirmation
+that the server has launched successfully.
+
+Query the server making sure to change the host address for each model:
+```json
+curl -X POST localhost:8000/v2/models/falcon7b/infer -d '{"inputs": [{"name":"prompt","datatype":"BYTES","shape":[1],"data":["How can you be"]}]}'
+curl -X POST localhost:8000/v2/models/mistral7b/infer -d '{"inputs": [{"name":"prompt","datatype":"BYTES","shape":[1],"data":["Where are you going"]}]}'
+curl -X POST localhost:8000/v2/models/persimmon8b/infer -d '{"inputs": [{"name":"prompt","datatype":"BYTES","shape":[1],"data":["Where is the nearest"]}]}'
 ```
+In our testing, these queries return the following parsed results:
+```bash
+# falcon7b
+"How can you be sure that you are getting the best deal on your car"
 
-Similarly, we can query our models specialized for text generation in this way:
+# mistral7b
+"Where are you going? I’m going to the beach."
 
-```bash
-python3 client.py -m opt-125m --prompt "Who am I?"
-python3 client.py -m gpt2 --prompt "Who am I?"
+# persimmon8b
+"Where is the nearest starbucks?"
 ```
-Which will return a result similar to:
+## 'Hour Zero' Support
```

Comment:
"Hour zero" was just me being cheeky. I think we should probably call it "day zero" or something more generic like "bleeding edge models" or something. CC @nnshah1 who probably has some thoughts

Reply:
I appreciated the cheekiness, but I guess we can change it to something more tame... :disappointed: